packages/@aws-cdk/aws-iam/lib/role.ts

```diff
@@ -148,6 +148,14 @@ export class Role extends Resource implements IRole {
   /**
    * Imports an external role by ARN.
    *
+   * If the imported Role ARN is a Token (such as a
+   * `CfnParameter.valueAsString` or a `Fn.importValue()`) *and* the referenced
```

Comment:
Should we use Token in our API docs? even with the parentheses this might be to much reading in the IDE